platform/source/lib/ax/platform/rest.py

```diff
@@ -228,7 +227,14 @@ def put_spot_instance_config():
 @_app.route("/v1/axmon/cluster/spot_instance_config", methods=['GET'])
 def get_spot_instance_config():
     response = requests.get(MINION_MANAGER_HOSTNAME + ":" + MINION_MANAGER_PORT + "/spot_instance_config")
-    return response.text
+    details = response.json()
```

Comment:
always do `response.raise_for_status()` and validate HTTP status code

Reply:
Nice. Done!